src/types/config-output.ts

```diff
@@ -0,0 +1,11 @@
+export type Platform = 'ios' | 'android';
+
+// disabling eslint here because these truly could be any
+/* eslint-disable */
+export type ConfigOutput = {
+    [key: string]: {
+        ios?: any;
+        android?: any;
+    }
```

Comment:
This block seems to be repeated multiple times in the code. Can we DRY it up?

Reply:
yes have created a separate `Env` type that contains these fields